ezgui/src/backend_glow.rs

```diff
@@ -4,90 +4,31 @@ use glow::HasContext;
 use std::cell::Cell;
 use std::rc::Rc;
 
-pub fn setup(window_title: &str) -> (PrerenderInnards, winit::event_loop::EventLoop<()>) {
```

Comment:
setup remains pretty distinct between wasm and native, so it's still duplicated.